ruby/datastax-v3/simple-connection/sf-class2-root.crt

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can we use the certificate generated specifically for use in samples here: https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/safenames.html#credentials